### PR TITLE
No js image display

### PR DESCRIPTION
**Pull request by gestchild**
Without js enabled the images were displaying weirdly on the site: duplicated/ blurry with alt text showing/ covering the whole page on works:

localhost_3000_stories
promos
localhost_3002_works_etk3phgd_query sun page 1

This fixes that

### CODE REVIEW

File: common/styles/components/_lazyload.scss [hunk 1]
```diff
@@ -2,6 +2,8 @@
   opacity: 0.6;
   transition: opacity $transition-properties;
 
+  display: none;
```
[the remark this answers is not in the document]
Comment (gestchild) [reply common/styles/components/_lazyload.scss]: I can't think of any other way of stopping 2 images from displaying in a no javascript environment, can you?

Of course the trade off is that they don't fade in any more when you do have javascript.

[the remark this answers is not in the document]
Comment (jamesgorrie) [reply common/styles/components/_lazyload.scss]: Maybe we should write up what we think is the best solution, and then implement it from there?

e.g.
- Resource page
  - Only show the main image at resonable res 800
  - Link through to all other images
- List page
  - Don't show images?

And then enhance the page to have the optimim image loaded in when scrolled into view (r there abouts)?

[the remark this answers is not in the document]
Comment (gestchild) [reply common/styles/components/_lazyload.scss]: Yep, definitely think we should do this, but one for another PR - would like to get this in as the site looks pretty broken with JS off at the moment

[the remark this answers is not in the document]
Comment (jamesgorrie) [reply common/styles/components/_lazyload.scss]: Okay, I'm good for this to go in if @davidpmccormick's concerns are aleaved.

**Pull request merged**